### DISCUSSION TITLE
Does it work with Bootstrap 5?

### DISCUSSION THREAD

**Discussion by ajayb11** [orestbida/cookieconsent#452] [Q&A] [answered]
We use Bootstrap 5 and I want to incorporate this consent popup in my app, however I don't want additional CSS.

**Comment by dombyte** [comment, marked as answer]
Well as I understand this plugin is not using Bootstrap Classes etc. so for the JS to work properly you need the CSS provided with the plug-in.
If you want you could change the whole plug-in to work with bootstrap but I would recommend using the provided CSS. It's very lightweight and minimalistic. Also I think the look is quite similar compared to bootstrap.
